docs/kqueen.rst

```diff
@@ -111,7 +113,7 @@ Example configuration files are located in `config/` directory. Default configur
       - Name of Jenkins job used to deploy cluster.
     * - JENKINS_USERNAME
       - None
-      - Username for Jenkins login.
+      - Optional. Default username for Jenkins login. Is not used in UI
```

Comment:
Optional. Using in case of Queen-API standalone, otherwise, possibly to define Jenkins credentials from UI side

Reply:
Using in case of Queen-API standalone and default, you still can set it in request directly. That's why I call it default

Reply:
'possibly to define' is wrong construction

Reply:
ok

Reply:
Optional. The default Jenkins username. Not used in the UI.
Maybe it's worth adding: You can define Jenkins credentials through the web UI?